source/fundamentals/collations.txt

```diff
@@ -4,5 +4,311 @@ Collations
+   :copyable: true
+
+   .. input::
+      :language: go
+
+      filter := bson.D{{"name", bson.D{{"$lt", "Infinite Jest"}}}}
+      cursor, err := coll.Find(context.TODO(), filter)
+
+      var results []bson.D
+      if err = cursor.All(context.TODO(), &results); err != nil {
+         panic(err)
+      }
+      for _, result := range results {
+         fmt.Println(result)
+      }
+
+   .. output::
+      :language: none
+      :visible: false
+
+      [{name Ça} {length 1138}]
+      [{name Cryptonomicon} {length 918}]
+      [{name Emma} {length 474}]
+
+Without specifying a default ``books`` collation, the ``Find()`` method would follow default
+binary collation rules to determine the ``name`` values that precede ``"Infinite Jest"``. These
+rules place words beginning with "Ç" after those beginning with "I". The output would resemble
+the following:
+
+.. code-block:: json
+   :copyable: false
+
+   [{name Cryptonomicon} {length 918}]
+   [{name Emma} {length 474}]
+
+See :ref:`golang-retrieve` for help with using the ``Find()`` method.
+
+.. _golang-index-collation:
+
+Set a Collation on an Index 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can apply a collation when you create a new index on a collection. The index stores
+an ordered representation of the documents in the collection so your MongoDB instance
+does not need to perform the ordering in-memory. 
+
+To use the index in an operation, your operation must use the same collation as the one
+specified in the index. Additionally, ensure that the operation is covered by the index that
+contains the collation. Set a collation through an ``IndexOptions`` object and call the
+``CreateOne()`` method with your options object as an argument.
+
+Example
+```````
+
+After creating the ``books`` collection and applying a default collation, you cannot change
+the collection's default collation. However, you can create an index for the collection with
+a different collation. 
+
+The following example uses the ``CreateOne()`` method to create an ascending index on the ``name``
+field and specifies a new collation with an ``"en_US"`` locale:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input::
+      :language: go
+
+       myCollation := &options.Collation{Locale: "en_US"}
+       opts := options.Index().SetCollation(myCollation)
+
+       indexModel := mongo.IndexModel{
+         Keys:    bson.D{{"name", 1}},
+         Options: opts,
+       }
+
+       name, err := coll.Indexes().CreateOne(context.TODO(), indexModel)
+       if err != nil {
+         panic(err)
+       }
+       fmt.Println("Name of Index Created: " + name)
+
+   .. output::
+      :language: none
+      :visible: false
+
+       Name of Index Created: name_1
+
+.. _golang-op-collation:
+
+Set a Collation on an Operation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Applying a collation to an operation overrides any default collation previously
+defined for a collection. If you apply a new collation to an operation that differs
```

Comment:
Issue: It took me two or three reading to realize the first and second sentence in this paragraph are about different things. Consider adding some sort of transition so that it is more clear that one is about a collection and one is about an index.

Reply:
I put everything about indexes in a separate paragraph.

Reply:
Excellent edit